### ISSUE TITLE
NTR: Autonomic-voluntary dissociation of facial movements (Cranial nerves)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7803]
New term request Autonomic-voluntary dissociation of facial movements (segal_191127202954): Selective paralysis of voluntary fascio-pharyngo-glosso-masticatory movements with preservation of reflexive and automatic functions.

**Comment by pnrobinson** [comment]
@MickeySegal
do you have a definition and more context? This is a relatiely broad description - https://www.medlink.com/articles/automatic-voluntary-dissociation

**Comment by MickeySegal** [comment]
https://www.neurology.org/doi/10.1212/wnl.0000000000002336
The selective paralysis of voluntary fascio-pharyngo-glosso-masticatory movements with preservation of reflexive and automatic functions is referred to as “autonomic-voluntary dissociation,” which is a distinctive feature of FCMS.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000306
- label: Autonomic-voluntary dissociation of facial movements
- parents: [Abnormal nervous system physiology (HP:0012638)](https://hpo.jax.org/app/browse/term/HP:0012638)
- pmid: PMID:23809752
- def: The selective paralysis of voluntary fascio-pharyngo-glosso-masticatory movements with preservation of reflexive and automatic functions is referred to as autonomic-voluntary dissociation, which is a distinctive feature of Anterior opercular syndrome (also known as Foix-Chavany-Marie syndrome).
- comment: The patient cannot open or close the mouth, protrude the tongue or close the eyes to orders, but can smile, cry, yell or yawn automatically. Ischemia is the most frequent cause and it is generally due to bilateral opercular lesions, but bilateral subcortical lesions have also been reported. In the anterior opercular syndrome the connections be-tween the motor cortex and the V, VII, IX, XII cranial nuclei and brainstem are interrupted bilaterally. Emotional and spontaneous movements, dependent on extrapyramidal connections, thalamus and hypothalamus, are preserved (automatic-voluntary dissociation).
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7803
- synonyms:
